Makefile

```diff
@@ -52,18 +52,20 @@ cross:
 	GOOS=windows $(MAKE) build BUILDTAGS="$(BUILDTAGS) $(BUILD_TAGS_WINDOWS_CROSS)"
 	GOOS=darwin $(MAKE) build BUILDTAGS="$(BUILDTAGS) $(BUILD_TAGS_DARWIN_CROSS)"
 
-tools: tools.timestamp
+tools: .install.gitvalidation .install.golangci-lint
 
-tools.timestamp: Makefile
-	@GO111MODULE="off" go get -u $(BUILDFLAGS) golang.org/x/lint/golint
+.install.gitvalidation:
+	if [ ! -x "$(GOBIN)/git-validation" ]; then \
```

Comment:
This is not quite a new issue, but what guarantees `GOBIN` is set?

Reply:
Added some checks to make it more robust